protocols/imap/src/main/java/org/apache/james/imap/api/process/ImapSession.java

```diff
@@ -233,4 +233,10 @@ default Username getUserName() {
             .map(MailboxSession::getUser)
             .orElse(null);
     }
+
+    default boolean isPlainAuthDisabledEventually() {
+        boolean isPlainAuthDisabled = !isPlainAuthEnabled();
```

Comment:
```suggestion
        isPlainAuthEnabled() -> isSSLRequired()
```

Please dis-ambiguate the code as well...

Reply:
I do not get it. Do you suggest changing the method name here? 

Is not that `isSSLRequired` return `auth.requireSSL` meanwhile `isPlainAuthEnabled` return `auth.plainAuthEnabled`? => that is 2 diff things